ergotree-ir/src/serialization/expr.rs

```diff
@@ -133,6 +134,7 @@ impl Expr {
                 ExtractAmount::OP_CODE => Ok(Expr::ExtractAmount(ExtractAmount::sigma_parse(r)?)),
                 OpCode::SELECT_FIELD => Ok(Expr::SelectField(SelectField::sigma_parse(r)?)),
                 OpCode::CALC_BLAKE2B256 => Ok(CalcBlake2b256::sigma_parse(r)?.into()),
+                OpCode::CALC_SHA256 => Ok(CalcSha256::sigma_parse(r)?.into()),
```

Comment:
```suggestion
                CalcSha256::OP_CODE => Ok(CalcSha256::sigma_parse(r)?.into()),
```
To reuse node op code “binding”. Plus, an eye catches it quickly to re-affirm op code to node association here.

Reply:
Thank you for your advice! Updated.